README.md

```diff
@@ -2,13 +2,9 @@
 
 [![GitHub Build Status](https://github.com/cisagov/terraformer-packer/workflows/build/badge.svg)](https://github.com/cisagov/terraformer-packer/actions)
 
-This is a generic skeleton project that can be used to quickly get a
-new [cisagov](https://github.com/cisagov) GitHub
-[Packer](https://packer.io) project started. This skeleton project
-contains [licensing information](LICENSE), as well as
-[pre-commit hooks](https://pre-commit.com) and
-[GitHub Actions](https://github.com/features/actions) configurations
-appropriate for the major languages that we use.
+This project can be used to create Amazon Machine Images (AMIs) that
+include the [Terraform](https://www.terraform.io/) infrastructure as
+code (IaC) tool from [HashiCorp](https://www.hashicorp.com/).
```

Comment:
I think it's also worth mentioning that Ansible is installed and available as well.

Reply:
Please see commit 58ddc29.

Reply:
Thanks!